### PR TITLE
Add taxonomy fields to campaign entity

### LINKED ISSUE #180: Campaign: Taxonomy Term fields
Depends on #172 

Update dosomething_campaign field feature with:
- Cause (can have multiple terms)
- Action Type (one term)

### PR DESCRIPTION
**Pull request by angaither**
Fixes #180

### CODE REVIEW

File: modules/dosomething/dosomething_taxonomy/dosomething_taxonomy.info [hunk 1]
```diff
@@ -8,6 +8,8 @@ dependencies[] = taxonomy
 dependencies[] = uuid
 dependencies[] = uuid_features
 features[features_api][] = api:2
+features[field_base][] = field_action_type
+features[field_base][] = field_cause
```
Comment (aaronschachter) [inline modules/dosomething/dosomething_taxonomy/dosomething_taxonomy.info:12]: Maybe this is a new features thing, but I don't see the field definition for any of these fields.  like the field.inc or anything like that.  Forgot to include them in the commit?

Comment (angaither) [inline modules/dosomething/dosomething_taxonomy/dosomething_taxonomy.info:12]: totally.

**Comment by angaither** [comment]
@aaronschachter should be better now! :shipit:

**Comment by aaronschachter** [comment]
`field_test` -- remove?

File: modules/dosomething/dosomething_campaign/dosomething_campaign.features.field_base.inc [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+ */
+
+/**
+ * Implements hook_field_default_field_bases().
+ */
+function dosomething_campaign_field_default_field_bases() {
+  $field_bases = array();
+
+  // Exported field_base: 'field_test'
+  $field_bases['field_test'] = array(
```
Comment (aaronschachter) [inline modules/dosomething/dosomething_campaign/dosomething_campaign.features.field_base.inc]: hiss

**Comment by aaronschachter** [comment]
Looks good!  Ship it

**Pull request merged**